packages/itwinui-css/src/dialog/dialog.scss

```diff
@@ -171,7 +171,8 @@ $iui-dialogue-resizable-min-height: 7.75rem; // 7.75rem = 124px = title bar heig
   padding: 0 var(--iui-size-m);
   overflow-y: auto;
   overflow-y: overlay;
-  max-height: calc(100vh - 6.75rem); // 6.75rem = 108px = button height + paddings + bottom bar
+  max-height: calc(100vh - 6.75rem); //6.75rem = 108px = button height + paddings + bottom bar
+  max-height: calc(100svh - 6.75rem);
```

Comment:
actually i think `dvh` will be better than `svh` because it will always be correct regardless of whether the browser ui is present

Reply:
Do you mean that we should use only `dvh`? Or changing `svh` to `dvh` and still keep the `vh`?

Reply:
the latter.

we need the fallback because `dvh` has same browser support as `svh`. https://caniuse.com/viewport-unit-variants

Reply:
Thanks, updated. Also tested the change like you suggested and it seems to be working. I don't see that bug anymore.